cli/module_registry.go

```diff
@@ -679,3 +684,78 @@ func createTarballForUpload(moduleUploadPath string, stdout io.Writer) (string,
 	}
 	return tmpFile.Name(), nil
 }
+
+func readModels(path string, logger golog.Logger) ([]ModuleComponent, error) {
+	parentAddr, err := os.MkdirTemp("", "viam-cli-test-*")
+	if err != nil {
+		return nil, err
+	}
+	defer vutils.UncheckedErrorFunc(func() error { return os.RemoveAll(parentAddr) })
+	parentAddr += "/parent.sock"
+
+	cfg := modconfig.Module{
+		Name:    "xxxx",
+		ExePath: path,
+	}
+
+	mgr := modmanager.NewManager(parentAddr, logger, modmanageroptions.Options{UntrustedEnv: false})
```

Comment:
@cheukt idea: i think it would be nice to add some SDK helpers in the future so binaries themselves could return this info with a commandline flag without having to be started by the modmanager.

Reply:
@zaporter-work filed [this](https://viam.atlassian.net/browse/RSDK-5314) a while ago; that might be similar to what you're requesting here.

Reply:
i'm not sure about this as then you'd have to add to every sdk
this seems fine to me